components/modbus_controller.rst

```diff
@@ -53,73 +68,73 @@ Technically there is no difference between the "inline" and the standard definit
 .. code-block:: yaml
 
     esphome:
-      name: solarstation
-      platform: ESP32
-      board: esp32dev
+        name: solarstation
+        platform: ESP32
+        board: esp32dev
```

Comment:
No need to ad extra spaces here? (Should be 2 spaces for yaml)

Reply:
right - the linter errors about the 4 spaces  confused me.
Branch is changed to next now